src/java.base/unix/native/libjli/java_md_solinux.c

```diff
@@ -422,6 +425,12 @@ CreateExecutionEnvironment(int *pargc, char ***pargv,
                         jrepath
                         );
 
+#ifdef AIX
+                if (__power_9_andup() && __power_nx_gzip()){
+                    sprintf(new_runpath + print_offset, ":" ZLIBNX_PATH);
+                }
```

Comment:
I think this should be contributed via a new `%s` parameter in the format, eliminating the need to define `print_offset` (a definition that isn't legal in C code where it appears). The matching value would be an empty string or `":" ZLIBNX_PATH`.

Please use `":" ZLIBNX_PATH` consistently (as opposed to `ZLIBNX_PATH ":"`).

Reply:
Fixed in cfb5b89. Any format comments?